sites/platform/src/development/ssh/troubleshoot-ssh.md

```diff
@@ -156,4 +156,4 @@ You can use this information to make one last check of the private key file.
 
 {{% troubleshoot %}}
 
-If you're still stuck, [submit a support ticket and provide the full SSH debug information](https://console.platform.sh/-/users/:user/tickets).
+If you're still stuck, [submit a support ticket and provide the full SSH debug information](https://console.platform.sh/-/users/:user/tickets).
```

Comment:
```suggestion
If you're still stuck, [submit a support ticket and provide the full SSH debug information](https://console.{{% vendor/urlraw "host" %}}/-/users/:user/tickets).
```

Reply:
I'm not sure this substitution will work, but we can always alternate a `version/specific` if not